src/main/java/com/eagrigorieva/operation/Operation.java

```diff
@@ -0,0 +1,28 @@
+package com.eagrigorieva.operation;
+
+import com.eagrigorieva.model.Task;
+import lombok.Data;
+import lombok.extern.log4j.Log4j2;
+
+import java.util.List;
+
+import static com.eagrigorieva.enumeration.TaskStatus.CREATED;
+
+@Data
+@Log4j2
+public abstract class Operation {
+    
+    protected void printTaskList(List<Task> taskList, String id) {
```

Comment:
Существенно ничего не поменялось. 
Посмотрел реализацию поля id, оно, оказывается, генерируется через UUID. Тогда, на мой взгляд, лучше переделать метод так, чтобы он на вход принимал Task и ее индекс.
```suggestion
    protected void printTaskList(int id, Task selectedTask) {
        log.debug("{}. [{}] {}", id, selectedTask.getTaskStatus() == CREATED ? "" : "x", selectedTask.getDescription());
        System.out.printf("%d. [%s] %s\n", id, selectedTask.getTaskStatus() == CREATED ? "" : "x", selectedTask.getDescription());
    }
```
В этом случае за передачу id будет ответственнен вызывающий код.

Reply:
Поправила